projects/robots/boston_dynamics/spot/controllers/Makefile

```diff
@@ -0,0 +1,23 @@
+# Copyright 1996-2020 Cyberbotics Ltd.
```

Comment:
This makefile should be added to this list:
https://github.com/cyberbotics/webots/blob/master/projects/robots/Makefile#L20